classes/PrestaShopAutoload.php

```diff
@@ -166,6 +166,15 @@ public function load($className)
      */
     public function generateIndex()
     {
+        if (defined('_PS_CREATION_DATE_')) {
+            $creationDate = _PS_CREATION_DATE_;
+            if (!empty($creationDate) && Configuration::get('PS_DISABLE_OVERRIDES')) {
+                $this->_include_override_path = false;
+            } else {
+                $this->_include_override_path = true;
+            }
```

Comment:
Is there a reason to add the `else` case, while the default value seems to be set before the __contrust?

Reply:
to be able to test it properly, otherwise in the same test it will never switch back to true. And it's more easier to understand that way

Reply:
We can probably set directly the `if` result in the `$this->_include_override_path` affectation, but I won't mind  if you don't do it.